CODEOWNERS

```
@@ -1 +1 @@
-* @armfazh @bhalleycf @captain-mota @chris-wood @cjpatton @lbaquerofierro- @mendess @will118
+* @armfazh @bhalleycf @captain-mota @cjpatton @lbaquerofierro- @mendess @will118
```

```suggestion
* @armfazh @bhalleycf @captain-mota @cjpatton @lbaquerofierro- @mendess
```

Will also no longer works at cloudflare